Spore ModAPI/Spore/Simulator/cCommunityLayout.h

```diff
@@ -12,7 +12,7 @@ namespace Simulator
 		/* 00h */	bool mIsOccupied;
 		/// A building, turret, ornament, etc
 		/* 04h */	cGameDataPtr mpObject;
-		/* 08h */	int field_8;
+		/* 08h */	int field_8; // 0
```

Comment:
These comments usually highlight the default value initialized in the constructor. 0 is assumed to be the default, so it is omitted